client/src/components/ImageContainer.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import styled, { css } from 'styled-components';
+import PropTypes from 'prop-types';
+
+function ImageContainer({ isLarge, movie }) {
+  const baseUrl = 'https://image.tmdb.org/t/p/original/';
+
+  return (
+    <>
```

Comment:
Your component <MovieContainer/> is already wrapping around the whole content so you do not need the additional React Fragments here

Reply:
You are right, thanks!